ocaml/xapi/importexport.ml

```diff
@@ -107,15 +107,15 @@ let this_version __context =
     This will get complicated over time... *)
 let assert_compatible ~__context other_version =
   let this_version = this_version __context in
-  (* error if major versions differ; also error if this host has a
-     lower minor vsn than the import *)
+  (* error if this host has a lower vsn than the import *)
   if
-    this_version.xapi_vsn_major <> other_version.xapi_vsn_major
-    || this_version.xapi_vsn_minor < other_version.xapi_vsn_minor
+    this_version.xapi_vsn_major < other_version.xapi_vsn_major
+    || this_version.xapi_vsn_major = other_version.xapi_vsn_major
```

Comment:
This relies on && having higher precedence than ||

Reply:
ocamlformat doesn't add parantheses here (they'd be redundant) but it does it show the precedence via indentation, so this should be fine.

Reply:
Yes, I actually had () around it, but `make format` removed them :)